client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -1203,9 +1203,23 @@
         };
     }
 
-    function ContentCreateDirective() {
+    ContentCreateDirective.$inject = ['templatesSettingsService'];
+    function ContentCreateDirective(templatesSettingsService) {
         return {
-            templateUrl: 'scripts/superdesk-authoring/views/sd-content-create.html'
+            templateUrl: 'scripts/superdesk-authoring/views/sd-content-create.html',
+            link: function(scope) {
+                scope.contentTemplates = null;
+
+                var fetchTemplates = function() {
```

Comment:
We can get rid of this private function here, as its only purpose is to be invoked once when the directive is linked. Just execute the function's body and that's it. :wink: